src/App.tsx

```diff
@@ -1,39 +1,75 @@
-import React from 'react';
+import { Component } from 'react';
+import CircularProgress from '@mui/material/CircularProgress';
+
+import Paper from '@mui/material/Paper';
 import './App.scss';
+import { Clock } from './Clock';
```

Comment:
I think it would be better to move Clock component to a separate Components folder.

Reply:
thanks, done